devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy

```diff
@@ -192,28 +183,26 @@ def zwaveEvent(physicalgraph.zwave.Command cmd) {
 }
 
 def on() {
-	[
-		zwave.basicV1.basicSet(value: 0xFF).format(),
-		zwave.switchBinaryV1.switchBinaryGet().format(),
-		"delay 3000",
-		zwave.meterV2.meterGet(scale: 2).format()
-	]
+	encapSequence([
+		zwave.basicV1.basicSet(value: 0xFF),
+		zwave.switchBinaryV1.switchBinaryGet(),
+		meterGet(scale: 2)
+	], 3000)
 }
 
 def off() {
-	[
-		zwave.basicV1.basicSet(value: 0x00).format(),
-		zwave.switchBinaryV1.switchBinaryGet().format(),
-		"delay 3000",
-		zwave.meterV2.meterGet(scale: 2).format()
-	]
+	encapSequence([
+		zwave.basicV1.basicSet(value: 0x00),
+		zwave.switchBinaryV1.switchBinaryGet(),
```

Comment:
Is there a reason you changed the delay between the set and get for this handler?